### ISSUE TITLE
Inversion of svg icons on contrast theme on native

### ISSUE THREAD

**Issue by lunars97** [digitalfabrik/integreat-app#3352]
### Describe the Problem
Once we set all the icons to svg (currently we have not replaced the PNGs to SVGs yet), we need to invert them on first level and second level to align with contrast theme like it is done on the web. There is a problem on react native with inversion of icon colors, 
### Describe Your Preferred Solution

So far I found one solution to use https://www.npmjs.com/package/react-native-color-matrix-image-filters

### Describe Possible Alternatives

Should be done on cms side to replace color with fill="currentColor" on SVGs path

### Related Issues

#1021

**Comment by osmers** [comment]
The related CMS issue is done - I'll remove the blocked label. In case it's not completely done, please re-add the label.
